recipe/pre-unlink.sh

```diff
@@ -1,4 +1,2 @@
-{
-  "${PREFIX}/bin/jupyter-nbextension" disable --sys-prefix --py nbgrader
-  "${PREFIX}/bin/jupyter-serverextension" disable --sys-prefix --py nbgrader
-} >>"$PREFIX/.messages.txt" 2>&1
+"${PREFIX}/bin/jupyter-nbextension" uninstall nbgrader --py --sys-prefix >> "${PREFIX}/.messages.txt" 2>&1
+"${PREFIX}/bin/jupyter-serverextension" disable nbgrader --py --sys-prefix >> "${PREFIX}/.messages.txt" 2>&1
```

Comment:
This was copied from https://github.com/conda-forge/nbpresent-feedstock initially, but seems it is not needed. I am guessing it is to reduce the repeated addition of piping each command to `messages.txt`